### PR TITLE
[MU4] Moved libcrashreporter to the main repo

### PR DESCRIPTION
**Pull request by igorkorsukov**
Moved libcrashreporter to the main repo

### CODE REVIEW

File: thirdparty/libcrashreporter/src/libcrashreporter-gui/CrashReporter.cpp [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+                m_ui->progressLabel->setText( tr( "We cannot gather useful debug information on your system." ) );
+                m_ui->button->setText( tr( "Close" ) );
+            }
+        });
+
+        m_btg->start();
+        m_ui->progressBox->setVisible( true );
+        m_ui->sendBox->setVisible( false );
+
+        m_ui->progressLabel->setText( tr( "Gathering debug information..." ) );
```
Comment (Jojo-Schmitz) [inline thirdparty/libcrashreporter/src/libcrashreporter-gui/CrashReporter.cpp]: So we can finally fix that `...` (triple dot) here to `…` (elipsis ;-))

**Comment by Jojo-Schmitz** [comment]
Readme.md should mention where it got cloned from (in the thirdparty section)

File: thirdparty/libcrashreporter/src/libcrashreporter-gui/CrashReporter.cpp (hunk 1 above)
Comment (igorkorsukov) [reply thirdparty/libcrashreporter/src/libcrashreporter-gui/CrashReporter.cpp]: fixed

**Comment by igorkorsukov** [comment]
> Readme.md should mention where it got cloned from (in the thirdparty section)

added

File: thirdparty/libcrashreporter/src/libcrashreporter-gui/CrashReporter.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply thirdparty/libcrashreporter/src/libcrashreporter-gui/CrashReporter.cpp]: Thanks, one sting less to 'translate' into en_US :-)

**Pull request merged**